setup.py

```diff
@@ -9,6 +9,9 @@ def getVersionFromFile():
 
 here = path.abspath(path.dirname(__file__))
 
+def getRequirements():
+    return open("docs/requirements.txt").readlines()
```

Comment:
Requirements for building the docs should go into the `project.optional-dependencies` section of `pyproject.toml`

Reply:
See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#dependencies-and-requirements and e.g.  https://github.com/cta-observatory/ctapipe/blob/970300e924382c3113cd36f4fa68b97fdf0a9730/pyproject.toml#L53

Reply:
We have the dependencies in the setup.py and in the dependencies section in the pyproject.toml file dependencies = [
    "dl1_data_handler>=0.12.0",
    "astropy",
    "matplotlib",
    "numpy",
    "pandas",
    "pip",
    "pyyaml",
    "scikit-learn",
    "ctaplot",
    "numba>=0.56.2,<0.57",
    "tensorflow>=2.15,<2.16",
    "pydot",
    "pyirf",
]

Reply:
I misunderstood then. I thought `docs/requirements.txt` are *additional* requirements that are needed to build the documentation (e.g. sphinx)